### PR TITLE
OSDOCS#7604: Updated link to Nutanix docs

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.11+

Issue:
This PR addresses [osdocs-7604](https://issues.redhat.com/browse/OSDOCS-7604).

Link to docs preview:

- Installing a cluster on Nutanix > [Configuring the default storage container](https://67804--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-nutanix-installer-provisioned#registry-configuring-storage-nutanix_installing-nutanix-installer-provisioned)
- Installing a cluster on Nutanix in a restricted network > [Configuring the default storage container](https://67804--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned#registry-configuring-storage-nutanix_installing-restricted-networks-nutanix-installer-provisioned)

QE review:
- [N/A] QE has approved this change.

No QE required. Updating a link to supporting Nutanix docs.

### CODE REVIEW

File: modules/registry-configuring-storage-nutanix.adoc [hunk 1]
```diff
@@ -6,4 +9,4 @@
 
 After you install the cluster, you must install the Nutanix CSI Operator and configure the default storage container for the cluster.
 
-For more information, see the Nutanix documentation for link:https://opendocs.nutanix.com/openshift/operators/csi/[installing the CSI Operator] and link:https://opendocs.nutanix.com/openshift/install/ipi/#openshift-image-registry-configuration[configuring registry storage].
+For more information, see the Nutanix documentation for link:https://opendocs.nutanix.com/openshift/operators/csi/[installing the CSI Operator] and link:https://opendocs.nutanix.com/openshift/post-install/[configuring registry storage].
```
Comment (ShaunaDiaz) [inline modules/registry-configuring-storage-nutanix.adoc:12]:
```suggestion
See link:https://opendocs.nutanix.com/openshift/operators/csi/[Nutanix CSI Operator, Installing the Operator] and link:https://opendocs.nutanix.com/openshift/post-install/[OpenShift Image registry configuration] for details.
```
Changed to be more literal. It took me a second to figure out where I should go. Easier with the exact linked titles.

**Review by ShaunaDiaz — APPROVED** [review]

File: modules/registry-configuring-storage-nutanix.adoc (hunk 1 above)
Comment (mjpytlak) [reply modules/registry-configuring-storage-nutanix.adoc:12]: Pretty sure our guidance requires us to use, "For more information". Going to leave as is.

**Pull request merged**